README.md

```diff
@@ -77,3 +77,11 @@ Your feedback is our motivation to move on. Please let us know your questions, c
 ## License
 
 [Apache License 2.0](https://github.com/sql-machine-learning/sqlflow/blob/develop/LICENSE)
+
+## Published
+
+- An arXiv paper at https://arxiv.org/abs/2001.06846
```

Comment:
Add markdown links to avoid copy-n-paste?

Reply:
I believe users can click the URL and go to the arXiv page.